packages/rocketchat-theme/client/imports/components/userInfo.css

```diff
@@ -176,6 +176,8 @@
 		&__item {
 			overflow: hidden;
 
+			max-width: 45%;
```

Comment:
Why is this better than adding `overflow: hidden` in the parent element (`.rc-user-info-action`)?

Reply:
It isn't! I changed 😄 now! Thank you 🙌